test/e2e-secondary-network/secondary_network_test.go

```diff
@@ -219,6 +237,204 @@ func (data *testData) pingBetweenInterfaces(t *testing.T) error {
 	return nil
 }
 
+// pauseAndResumeAntreaAgent pauses or resumes the Antrea agent DaemonSet by modifying its update strategy.
+func (data *testData) pauseAndResumeAntreaAgent(namespace, daemonsetName, action string) error {
```

Comment:
@antoninbas : do you have a suggestion on the right way to stop agent and start agent? We do not have existing code for these?